### ISSUE TITLE
[Bug Report] App crash at launch

### ISSUE THREAD

**Issue by j-px** [SapuSeven/BetterUntis#195]
<details>
<summary>Logs</summary>

```
java.lang.RuntimeException: java.lang.reflect.InvocationTargetException
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:612)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1130)
Caused by: java.lang.reflect.InvocationTargetException
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:602)
	... 1 more
Caused by: java.lang.reflect.InvocationTargetException
	at java.lang.reflect.Constructor.newInstance0(Native Method)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:343)
	at com.sapuseven.untis.viewmodels.PeriodDataViewModel$c.a(Unknown Source:41)
	at o.n.d0.getValue(:4)
	at d.a.a.i.c.B0(Unknown Source:2)
	at d.a.a.i.c.Q(Unknown Source:11)
	at androidx.fragment.app.Fragment.h0(Unknown Source:15)
	at o.l.d.r.U(:29)
	at o.l.d.r.S(:1)
	at o.l.d.r.T(Unknown Source:47)
	at o.l.d.r.v(Unknown Source:9)
	at o.l.d.e.onStart(:2)
	at o.b.k.l.onStart(Unknown Source:0)
	at d.a.a.b.b.onStart(Unknown Source:0)
	at android.app.Instrumentation.callActivityOnStart(Instrumentation.java:1435)
	at android.app.Activity.performStart(Activity.java:8231)
	at android.app.ActivityThread.handleStartActivity(ActivityThread.java:3853)
	at android.app.servertransaction.TransactionExecutor.performLifecycleSequence(TransactionExecutor.java:221)
	at android.app.servertransaction.TransactionExecutor.cycleToPath(TransactionExecutor.java:201)
	at android.app.servertransaction.TransactionExecutor.executeLifecycleState(TransactionExecutor.java:173)
	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:97)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2317)
	at android.os.Handler.dispatchMessage(Handler.java:106)
	at android.os.Looper.loop(Looper.java:246)
	at android.app.ActivityThread.main(ActivityThread.java:8595)
	... 3 more
Caused by: java.lang.IllegalArgumentException: Parameter specified as non-null is null: method com.sapuseven.untis.viewmodels.PeriodDataViewModel.<init>, parameter user
	at com.sapuseven.untis.viewmodels.PeriodDataViewModel.<init>(Unknown Source:2)
	... 28 more

```
</details>

**Additional information**

- Android version: _11_
- BetterUntis version: _3.4.5 (29)_
- Installation source: _com.android.vending_

**Comment by j-px** [comment]
Randomly had the same bug again today, and fixed it by just clearing the app's data & logging in again. Weirdly enough it cleared around 200 MB of data though, not sure how those piled up!

**Comment by SapuSeven** [comment]
This will be fixed with the Material 3 Update (#290)
